app/blog/20161004-how-to-update-wordpress.static.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to update WordPress
+author:  Jenny Knuth
+posted: 20161004
+summary:
+  Steps to take to update and test a theme update in WordPress.
+_options:
+  layout: app/layouts/blog.static.hbs
+---
+
+## For the Developer:
+
+Before updating a WordPress theme on a live site, follow these steps to update your local install and test:
+
+Check for n.io-specific customizations in the .php files on the child theme. To do this, look at the existing .php file in the parent (e.g. Divi) theme and compare it to the same .php file in the child (e.g. nio) theme. Any changes in the child theme are customizations that may or may not still be needed. Evaluate these changes and make a patch with the customizations that are important to keep.
+
+Download the latest version of the theme. Replace the parent theme in your local install with the new theme folder. Share this new folder with the dev team so they can update their local installs (unlike the child theme folder, the parent folder is not a github repo—alternatively devs can grab the most recent install from siteground…discuss…). This parent-theme folder will be the new base from which to compare future changes.
+
+Make a new branch and update your child theme by replacing the child .php files with the up-to-date parent theme .php files and then adding the custom patch. It is a good idea to commit after updating each file.
```

Comment:
There may be a simpler way to go about this. I don't think there is anything you need to change about this content right now but once I am back in the office we can walk through an example and see if the entire parent/child theme update can be streamlined.